py/jpy-config/src/main/java/io/deephaven/jpy/PropertiesFromSubprocess.java

```diff
@@ -0,0 +1,61 @@
+package io.deephaven.jpy;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+class PropertiesFromSubprocess {
+
+    static Properties properties(String pythonName, Duration timeout)
+            throws IOException, InterruptedException, TimeoutException {
+        final Process process;
+        try (final InputStream in = PropertiesFromSubprocess.class.getResourceAsStream("introspect.py")) {
+            if (in == null) {
+                throw new IllegalStateException("Expected to find introspect.py resource");
+            }
+            // The "-" means that python will read script from stdin
+            try {
+                process = new ProcessBuilder(pythonName, "-").start();
+            } catch (IOException e) {
+                if (e.getMessage().contains("error=2, No such file or directory")) {
```

Comment:
This looks unreliable.
Maybe a case-insensitive check for "no such" ? Also a check for FNFE?

Reply:
I think maybe just re-wrapping the IOException makes sense.

Reply:
Testing on mac/windows seems important to make sure we get the same error. This code will run (that is, "fail") cleanly without jpy being built for the platform, so we don't even need to do that work.